src/core/hle/service/sockets/sockets.h

```diff
@@ -19,6 +19,7 @@ enum class Errno : u32 {
     INVAL = 22,
     MFILE = 24,
     PIPE = 32,
+    CONNREFUSED = 61,
```

Comment:
```suggestion
    CONNREFUSED = 111,
```
Where did you get 61?

Reply:
Interesting, i got 61 from the FreeBSD manual, but looking around i see most other places using 111 for CONNREFUSED.

Reply:
The errno constants used in the Switch come from Android and are therefore Linux constants.